src/api/emails/index.ts

```diff
@@ -119,14 +119,14 @@ const passwordResetLink = async (urlOrigin: string, emailAddress: string, name:
 
 const applicationSubmitted = {
   /**
-   * applicationSubmitted.account
+   * applicationSubmitted.applicationSubmittedEmail
    * Send "application submitted" email to an account
    * @param {Object} ApplicationSubmissionEmailVariables
    * @returns {Object} callNotify response
    */
-  account: async (variables: ApplicationSubmissionEmailVariables): Promise<EmailResponse> => {
+  applicationSubmittedEmail: async (variables: ApplicationSubmissionEmailVariables): Promise<EmailResponse> => {
     try {
-      console.info('Sending application submitted email to account');
+      console.info('Sending application submitted email to business owner or provided contact');
```

Comment:
Naming of this function feels strange, `applicationSubmitted.applicationSubmittedEmail`, how about renaming the parent object and this function so it reads like this?

```js
application.submittedEmail()
```

Also should the log say "business owner and/or provided contact"?

Reply:
I kept it as or since the controller sends it to one at a time